app/config/sources.inc.php

```diff
@@ -363,6 +363,12 @@
         'priority'          => 2,
         'supported_locales' => $mozillaorg,
     ],
+    'mozorg/about/governance/policies/participation.lang' => [
+        'priority'          => 3,
```

Comment:
No need to add priority 3, it's the default for mozilla.org files
https://github.com/mozilla-l10n/langchecker/blob/master/app/config/sources.inc.php#L103